Golang/json/postreq.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type postData struct {
+	Name      string
+	Job       string
+	Id        string
+	CreatedAt string
+}
+
+func postreq() {
+
+	fmt.Println("\nPost req Function")
+	details, _ := json.Marshal(map[string]string{
+		"name": "rithish",
+		"job":  "developer",
+	})
+
+	response, err := http.Post("https://reqres.in/api/users", "application/json", bytes.NewBuffer(details))
```

Comment:
You can put the values of the urls into a constant and place them in a separate file.